src/module/chat/cards.ts

```diff
@@ -21,7 +21,7 @@ export class IronswornChatCard {
 		const moveLinks = html.find('a[draggable]')
 		const maybeTablePromises = moveLinks.map(async (_i, el) => {
 			const { pack, id } = el.dataset
-			if (!pack || !id) return []
+			if (pack == null || id == null) return []
```

Comment:
Bleh I keep forgetting `undefined == null` yields `true`. This language!

Reply:
yeah, that trips me up as well. some part of my brain sees `==` and expects a positive result. on the other hand, it's probably safer that way, so maybe i just need to get used to it :shrug: 

i could take or leave that rule, so if ditching it would make your life easier, go for it, IMO